### ISSUE TITLE
Cannot add local sources

### ISSUE THREAD

**Issue by savannahjune** [mapbox/mapbox-studio-classic#795]
When I update to the latest version yesterday, I lost the ability to choose and upload local sources. I have data in the mapbox studio folder locally, but there is no option to add it. Remote sources are behaving normally. Perhaps I'm missing something, but I recall adding local sources from this screen before. 

Thanks :)

![upload issue](https://cloud.githubusercontent.com/assets/2073292/4194235/3f524d08-37ab-11e4-901d-61d7f79df3b4.png)

**Comment by yhahn** [comment]
@savannahjune hmm... did you upgrade from pre 0.0.4? There was a config directory change from 0.0.4 on:

https://github.com/mapbox/mapbox-studio/blob/mb-pages/CHANGELOG.md#004

You should be able to repopulate Mapbox Studio's history of your local sources by browsing for them and opening them up.

**Comment by savannahjune** [comment]
@yhahn No, I updated from the previous version.  I haven't lost any project's sources, as I saved them in a separate place when updating, but was unable to add any new local sources.  However, since then this issue has become more pressing: https://github.com/mapbox/mapbox-studio/issues/837 This can rest for a while since it probably won't persist in the latest version once I get it running.
